src/send.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Send
+ * Communicate an action to stores
+ */
+
+module.exports = function send (store, state, { payload, type }) {
+  if (type in store) {
+    return store[type].call(store, state, payload)
+  }
```

Comment:
This conditionally allows stores that implement the "special" methods to continue to work. In Microcosm 10.0.0 I will remove them (and we'll add a deprecation working on the 9.0.0 branch)